packages/quick_actions/quick_actions_ios/ios/quick_actions_ios.podspec

```diff
@@ -14,9 +14,11 @@ Downloaded by pub (not CocoaPods).
   s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
   s.source           = { :http => 'https://github.com/flutter/plugins/tree/main/packages/quick_actions' }
   s.documentation_url = 'https://pub.dev/packages/quick_actions'
-  s.source_files = 'Classes/**/*'
-  s.public_header_files = 'Classes/**/*.h'
+  s.source_files = 'Classes/**/*.{h,m}'
+  s.public_header_files = 'Classes/*.h'
```

Comment:
Doesn't this also match `Classes/PrivateHeaders/*.h`?

Reply:
Oh I removed `/**`, so it won't include the private headers.

Reply:
See this: 

![Screen Shot 2022-08-04 at 4 43 10 PM](https://user-images.githubusercontent.com/41930132/182973428-57e3c2e5-35eb-4b2b-a5cb-77c40cf68b9c.png)